packages/arb-token-bridge-ui/src/util/EthDepositUtils.ts

```diff
@@ -68,24 +68,29 @@ export async function depositEthEstimateGas(
   })
 
   if (isDifferentDestinationAddress) {
-    const depositToRequest = await ethBridger.getDepositToRequest({
-      amount,
-      from: address,
-      parentProvider: parentChainProvider,
-      childProvider: childChainProvider,
-      // we know it's defined
-      destinationAddress: String(destinationAddress)
-    })
-
-    const estimatedParentChainGas = await parentChainProvider.estimateGas(
-      depositToRequest.txRequest
-    )
-
-    return {
-      estimatedParentChainGas,
-      estimatedChildChainGas: depositToRequest.retryableData.gasLimit,
-      estimatedChildChainSubmissionCost:
-        depositToRequest.retryableData.maxSubmissionCost
+    try {
+      const depositToRequest = await ethBridger.getDepositToRequest({
+        amount,
+        from: address,
+        parentProvider: parentChainProvider,
+        childProvider: childChainProvider,
+        // we know it's defined
+        destinationAddress: String(destinationAddress)
+      })
+
+      const estimatedParentChainGas = await parentChainProvider.estimateGas(
+        depositToRequest.txRequest
+      )
+
+      return {
+        estimatedParentChainGas,
+        estimatedChildChainGas: depositToRequest.retryableData.gasLimit,
+        estimatedChildChainSubmissionCost:
+          depositToRequest.retryableData.maxSubmissionCost
+      }
+    } catch {
```

Comment:
```suggestion
    } catch (error) {
```

for Sentry
see https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1182

Reply:
updated